14/umbraco-commerce/tutorials/getting-started-with-commerce.md

```diff
@@ -0,0 +1,229 @@
+---
+description: "A guide to setting up Umbraco Commerce, including an introduction to basic concepts and configuration."
+---
+# A Beginner's Guide to Umbraco Commerce
+
+This tutorial focuses exclusively on setting up Umbraco Commerce, introducing key concepts, and the configuration process. It has been tested on the latest releases of **Umbraco CMS version 14** and **Umbraco Commerce version 14**.
+
+{% hint style="info" %}
+This tutorial does not provide instructions for building a complete eCommerce website.
```

Comment:
I know something similar is mentioned above, but just to make it even clearer - how about adding something like this?

```suggestion
This tutorial does not provide instructions for building a complete eCommerce website. It focusses solely on the setup through the Umbraco backoffice.
```